src/chrome/content/rules/Orion_Magazine.org.xml

```diff
@@ -1,10 +1,19 @@
+<!--
+
+	Problematic domains:
+
+	- orionmagazine.com (timeout)
+
+-->
 <ruleset name="Orion Magazine.com">
 
 	<target host="orionmagazine.org" />
 	<target host="www.orionmagazine.org" />
+	<target host="orionmagazine.com" />
+	<target host="www.orionmagazine.com" />
 
 
-	<rule from="^http://(www\.)?orionmagazine\.com/"
-		to="https://$1orionmagazine.com/" />
+	<rule from="^http://(?:www\.)?orionmagazine\.(?:org|com)/"
```

Comment:
Please avoid non-capturing groups

Reply:
I dont understand why you are wanting me to avoid non-capturing groups, but in any case removing orionmagazine.com as requested heavily modified this line.

Is it still problematic.  If so, why?

Reply:
For simple rules [we prefer](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#regex-conventions) something like
```xml
<!-- Certificate not valid for www -->
<rule from="^http://www\.example\.com/" to="https://example.com/" />
<rule from="^http:" to="https:" />
```

but in this case `<rule from="^http:" to="https:" />` seems to [work just fine](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#snapping-redirects).